docs/assignment1.md

```diff
@@ -1,62 +1,154 @@
+
+Word Count: 552
 
 ## Features
-Author(s): `name of the team member(s) responsible for this section`
-
-<When defining both functional features and quality requirements, remember that you will need to come back to them in Assignments 2 and 3 and explicitly state how specific parts of models/implementation satisfy them.>
+Author(s): Koen van den Burg
 
 ### Functional features
 
-As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
-
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+These functional features are imperative in order to make the game work the way is necessary. Many of these features are inherently linked with each other, therefore if one feature is not met in the final product, then the entire game falls apart. These features are inspired by what we know of adventure games, as well as certain features that are present in text-based games only, such as written commands and a turn-based time mechanic.
+
+<table>
+<tr>
+<th>ID</th>
+<th>Short Name</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>F1</td>
+<td>Commands</td>
+<td>Game shall have a <b>text-based option system</b>, where the player can type different commands in a command prompt which will result in different
+actions(F3).</td>
+</tr>
+<tr>
+<td>F2</td>
+<td>Interface</td>
+<td>The game shall be played inside a command prompt. Everything is text. Some scenarios should have ASCII art.</td>
+</tr>
+<tr>
+<td>F3</td>
+<td>Actions</td>
+<td>Typing different commands shal result in different actions in the game world, examples of such are:
+<ul>
+<li>Movement</li>
+<li>Attacking</li>
+<li>Conversation</li>
+<li>Interacting with items (doors, keys, picking up items)</li>
+<li>Dying (restarts the game)</li>
+<li>Healing</li>
+<li>Examining items and surrounding</li>
+<li>Sleeping</li>
+</ul>
+</td>
+</tr>
+<tr>
+<td>F4</td>
+<td>Items</td>
+<td>There shall be a multitude of different items which can be added to the inventory of the player (F8):
+<ul>
+<li>Weapons</li>
+<li>Keys</li>
+<li>Healing items</li>
+<li>Ammunition</li>
+</ul>
+</td>
+</tr>
+<tr>
+<td>F5</td>
+<td>Characters</td>
+<td>The game shall have characters, each with different statistics(F6) defining what they are (enemies, friendlies, etc).</td>
+</tr>
+<tr>
+<td>F6</td>
+<td>Statistics</td>
+<td>The gameshall have <b>statistics</b> for the player and other characters in the game. Statistics will be:
+<ul>
```

Comment:
"Attributes" would describe what you are trying to do better than the word "statistics".